javascripts/tagmanager.js

```diff
@@ -1635,6 +1635,15 @@
                 throwError: throwError,
                 Container: Container,
                 addContainer: function (containerConfig, templates) {
+                    var setDebugFlag = urlHelper.getQueryParameter('setDebugFlag');
```

Comment:
btw we would probably want to prefix that URL parameter with `mtm` like `mtmSetDebugFlag`

Reply:
BTW we also need to exclude this parameter from being tracked. Otherwise this URL parameter would appear in Matomo reports like `https://mydomain.com/path?mtmSetDebugFlag=1`. We can define such parameters to be excluded.

Reply:
updated and excluded